lib/publiccloud/instance.pm

```diff
@@ -69,7 +73,7 @@ sub scp {
     my $ssh_cmd = sprintf('scp %s -i "%s" "%s" "%s"',
```

Comment:
I know you are not using this function in this test, but there is an issue I found regarding quotes (see PR#7813).